### ISSUE TITLE
FileIO Integation / AstroImages.jl

### ISSUE THREAD

**Issue by sefffal** [JuliaAstro/FITSIO.jl#183]
Hi all,
I've been working on a [PR to AstroImages.jl](https://github.com/JuliaAstro/AstroImages.jl/pull/30) which relates to a few issues here:
* https://github.com/JuliaAstro/FITSIO.jl/issues/160
* https://github.com/JuliaAstro/FITSIO.jl/issues/18
* https://github.com/JuliaAstro/FITSIO.jl/issues/90

Once merged, I would like to register AstroImages with FileIO so that users can `load("abc.fits")` and get an AstroImage but I see that FITSIO is already registered : https://github.com/JuliaIO/FileIO.jl/blob/f0e5b9568d684bb4c6851e1b737b9a344d22504c/src/registry.jl#L467

Adding AstroImages as a second format would override `load` when the user has both packages installed (and maintain the status-quo if the user only has FITSIO installed). 
The FileIO integration with FITSIO does not appear to be documented so hopefully this wouldn't break anyone's workflow.

Basically I wanted to check with the FITSIO maintainers that registering a second package that can read/write FITS files won't break anyone's workflow.

Thanks!

Edit the WIP docs for the update to AstroImages are here for now: https://sefffal.github.io/AstroImages.jl/dev/

**Comment by sefffal** [comment]
Opened a PR with FileIO: https://github.com/JuliaIO/FileIO.jl/pull/363

This also adds support for `load("fits.gz")` which previously fell back on Libz.jl.
